.ci/pytorch/test.sh

```diff
@@ -1383,14 +1383,17 @@ test_executorch() {
   assert_git_not_dirty
 }
 
-test_linux_aarch64(){
-  python test/run_test.py --include test_modules test_mkldnn test_mkldnn_fusion test_openmp test_torch test_dynamic_shapes \
-       test_transformers test_multiprocessing test_numpy_interop --verbose
+test_linux_aarch64() {
+  if [[ $SHARD_NUMBER" == 1 ]]; then
+      python test/run_test.py --include test_modules test_mkldnn test_mkldnn_fusion test_openmp test_torch test_dynamic_shapes \
+           test_transformers test_multiprocessing test_numpy_interop --verbose
+  fi
 
   # Dynamo tests
   python test/run_test.py --include dynamo/test_compile dynamo/test_backends dynamo/test_comptime dynamo/test_config \
        dynamo/test_functions dynamo/test_fx_passes_pre_grad dynamo/test_interop dynamo/test_model_output dynamo/test_modules \
-       dynamo/test_optimizers dynamo/test_recompile_ux dynamo/test_recompiles --verbose
+       dynamo/test_optimizers dynamo/test_recompile_ux dynamo/test_recompiles \
+       --shard "$((SHARD_NUMBER-1))" "$NUM_TEST_SHARDS" --verbose
```

Comment:
@clee2000 do you know if using sharded runs twice will mess up the shard runtime computation?